modules/virt-add-disk-to-vm.adoc

```diff
@@ -38,6 +38,10 @@ Use this procedure to add a virtual disk to a {object}.
 
 . In the *Add Disk* window, specify the *Source*, *Name*, *Size*, *Type*, *Interface*, and *Storage Class*.
 
+.. Optional: Select the *Enable preallocation* checkbox if you want to use the disk's full space capacity immediately. However, this option should only be used if you really need the full capacity.
```

Comment:
in the backend docs they say:
https://docs.openshift.com/container-platform/4.8/virt/virtual_machines/virtual_disks/virt-using-preallocation-for-datavolumes.html

"Select the *Enable preallocation* checkbox to improve write performance when creating data volumes."

dose that makes sense ?

Reply:
I think it makes sense reading that section you provided. What about the 2nd sentence about needing full capacity? I thought about changing it to "However, this option should only be used if you really need maximum write performance." Or do you think it's not needed in the first place?